docs/manage_users.md

```diff
@@ -0,0 +1,129 @@
+# Manage users in Percona Everest
+
+Percona Everest provides user management capabilities, enabling you to securely log in through either the Percona Everest UI or the API.
+
+Local user management involves administering Percona Everest users to ensure secure access to database resources. This encompasses tasks such as creating and deleting users, updating their passwords, etc.
+
+When you install Percona Everest, an `admin` user is automatically created, granting full access to the system.
+
+!!! note alert alert-primary "Note"
+
+    It is recommended that you use the admin account for the initial configuration and subsequently transition to Single Sign-On (SSO) integration.
```

Comment:
Would be nice to link a reference to the SSO documentation page, once added

Reply:
Sure.

Reply:
Done